arns/arns-token-1.md

```diff
@@ -37,9 +38,11 @@ However, developers are not restricted to using Lua exclusively when building ne
 
 The **ARNS-TOKEN-1** Specification includes the following requirements:
 
-- Must have a `Name` which is a friendly nickname of this ANT.
-- Must have a `Ticker` which is a short token symbol, shown in block explorers and marketplaces.
-- Must have a `Logo` that is an Arweave Transaction ID that contains an icon used by downstream apps.
+- Should have a `Name` which is a friendly nickname of this ANT.
+- Should have a `Ticker` which is a short token symbol, shown in block explorers and marketplaces.
+- Should have a `Logo` that is an Arweave Transaction ID that contains an icon used by downstream apps.
+- Should have a `Description` that is a brief description of this ANT and its purpose or use.
+- Should have a table of `Keywords` used to further describe this ANT.
```

Comment:
It was mentioned somewhere putting limits on these. 512 or something for description and n keywords with max 32 chars per word.

Reply:
I had this detail down where the `Set-Description` and `Set-Keywords` handlers are.  Added it to this section as well.